functorch/functorch/_src/eager_transforms.py

```diff
@@ -839,7 +839,7 @@ def safe_unflatten(tensor, dim, shape):
     return tensor.unflatten(dim, shape)
 
 
-def jacfwd(func: Callable, argnums: argnums_t = 0, has_aux: bool = False):
+def jacfwd(func: Callable, argnums: argnums_t = 0, has_aux: bool = False, randomness: str = "error"):
```

Comment:
Let's start making arguments kwarg only. Does `*, randomness: str = "error"` work?

Reply:
Fixed and tests work